variants/STM32MP157_DK/variant.h

```diff
@@ -0,0 +1,225 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2017, STMicroelectronics
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ * 3. Neither the name of STMicroelectronics nor the names of its contributors
+ *    may be used to endorse or promote products derived from this software
+ *    without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *******************************************************************************
+ */
+
+#ifndef _VARIANT_ARDUINO_STM32_
+#define _VARIANT_ARDUINO_STM32_
+
+#ifdef __cplusplus
+extern "C" {
+#endif // __cplusplus
+
+/*----------------------------------------------------------------------------
+ *        Pins
+ *----------------------------------------------------------------------------*/
+
+#define PE7   0 // - USART7_RX
+#define PE8   1 // - USART7_TX and TIM1_CH1N
+#define PE1   2
+#define PD14  3 // - TIM4_CH3
+#define PE10  4 // - TIM1_CH2N
+#define PD15  5 // - TIM4_CH4
+#define PE9   6 // - TIM1_CH1
+#define PD1   7
+#define PG3   8
+#define PH6   9 // - TIM12_CH1
+#define PE11  10 // - SPI4_NSS and TIM1_CH2
+#define PE14  11 // - SPI4_MOSI and TIM1_CH4
+#define PE13  12 // - SPI4_MISO and TIM1_CH3
+#define PE12  13 // - SPI4_SCK and TIM1_CH3N
+#define PA12  14 // - I2C5_SDA
+#define PA11  15 // - I2C5_SCL and TIM1_CH4
+// CN2 Raspberry Pi GPIO connectors
```

Comment:
Maybe this could be cleaned

Reply:
The commented headers will work when a proper Linux device tree is used and enable the peripherals in PeripheralPins.c. The purpose of this PR is to push my idea of running RPi code on Linux and Arduino code in the same chip, so I disabled the RPi headers for now. I left it for someone (or me) who want to use Raspberry Pi headers in Arduino code.

Reply:
Ok